build.py

```diff
@@ -13,5 +13,9 @@
     os.system("conan package .. --build_folder=../")
     print(os.listdir("."))
     print(os.listdir("./lib"))
+    os.system("cd .. && conan export user/channel") # This will improve next 0.25 with an --cwd=.. arg
+    os.system("conan package_files Test/0.1@user/channel -f")  # -f in case it exists, overwrite
+    os.system("conan search Test/0.1@user/channel") # To show the package binary
+    # os.system("conan upload Test/0.1@user/channel --all -r=")
```

Comment:
Noted about using profiles, thanks!

Since we want the header files and libs visible in the client's workspace, I ended up using imports like so -

    [imports]
      include, *.h -> ./include # Copies all header files from packages include folders to my "include" folder
      lib, *.a -> ./lib         # Copies all static libraries from packages lib folders to my "lib" folder